toolset/run-tests.py

```diff
@@ -149,9 +145,6 @@ def main(argv=None):
     parser.add_argument('--list-tests', action='store_true', default=False, help='lists all the known tests that can run')
 
     # Benchmark options
-    parser.add_argument('--concurrency-levels', default=[8, 16, 32, 64, 128, 256], help='Runs wrk benchmarker with different concurrency value (type int-sequence)', action=StoreSeqAction)
-    parser.add_argument('--query-levels', default=[1, 5,10,15,20], help='Database queries requested per HTTP connection, used during query test (type int-sequence)', action=StoreSeqAction)
```

Comment:
With this, `concurrency_levels` and `query_levels` will no longer be configurable, right?  If so, how about removing those lines from `benchmark.cfg.example`?